.github/workflows/conformance-clustermesh.yaml

```diff
@@ -145,7 +145,7 @@ jobs:
             cm-auth-mode-2: 'cluster'
 
           - name: '7'
-            tunnel: 'vxlan'
+            tunnel: 'geneve'
```

Comment:
Aren't we losing VXLAN & Wireguard coverage?

Reply:
Hmm, you are right. I'd personally say that it shouldn't matter too much (I mean, the goal here is not to cover all possible combinations, as at the datapath level there should be practically no difference between single and multicluster traffic, and they are already covered in Conformance E2E). But let me revert one of the two changes.

Reply:
Looks like `Conformance clustermesh` workflow doesn't take too much time.
So, maybe instead of reverting you can add a new matrix test case?

Reply:
Not sure if that's really worth honestly. I'd not expect any difference between vxlan and geneve in this respect.